src/subscribables/subscribable.js

```diff
@@ -75,6 +79,20 @@ var ko_subscribable_fn = {
                 ko.dependencyDetection.end(); // End suppressing dependency detection
             }
         }
+        // Notify subscribers to the ALL_EVENT event, but not if that was the event that was fired (prevent double events)
+        if (this.hasSubscriptionsForEvent(allEvent) && event != allEvent) {
```

Comment:
The code duplication is definitely unfortunate and is something I'd refine before this was merged. For consistency, if `notifySubscribers` passed a third callback argument, we could swap the event and third param in the case of a notify subscribers with an 'all' event if you think that would be less confusing. But there's no way around passing the original event - if you subscribe to all events, you need to know what the original event was so you can react accordingly.

Either way, it seems like a onNoify hook is the direction this is heading so it's probably a bit of a moot point.